indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -118,7 +118,15 @@ def _process(self):
                 _('Download Attachments'),
                 'attachment',
                 type='href-custom',
-                url=url_for('.registrations_attachments_export', regform),
+                url=url_for('.registrations_attachments_export', regform, flatten=False),
+                weight=60,
+                extra_classes='js-submit-list-form regform-download-attachments',
+            ),
+            ActionMenuEntry(
+                _('Download Attachments (flat)'),
+                'attachment',
+                type='href-custom',
+                url=url_for('.registrations_attachments_export', regform, flatten=True),
                 weight=60,
```

Comment:
I'd avoid having the same weight on two entries:
```suggestion
                weight=59,
```

Reply:
Done!